src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse42.cs

```diff
@@ -18,115 +18,242 @@ public static class Sse42
         /// <summary>
         /// int _mm_cmpistra (__m128i a, __m128i b, const int imm8)
         ///   PCMPISTRI xmm, xmm/m128, imm8
-        /// int _mm_cmpistrc (__m128i a, __m128i b, const int imm8)
-        ///   PCMPISTRI xmm, xmm/m128, imm8
-        /// int _mm_cmpistro (__m128i a, __m128i b, const int imm8)
-        ///   PCMPISTRI xmm, xmm/m128, imm8
-        /// int _mm_cmpistrs (__m128i a, __m128i b, const int imm8)
+        /// </summary>
+        public static bool CompareImplicitLengthNotCAndNotZ(Vector128<sbyte> left, Vector128<sbyte> right, StringComparisonMode mode) => CompareImplicitLengthNotCAndNotZ(left, right, mode);
+
+        /// <summary>
+        /// int _mm_cmpistra (__m128i a, __m128i b, const int imm8)
         ///   PCMPISTRI xmm, xmm/m128, imm8
-        /// int _mm_cmpistrz (__m128i a, __m128i b, const int imm8)
+        /// </summary>
+        public static bool CompareImplicitLengthNotCAndNotZ(Vector128<byte> left, Vector128<byte> right, StringComparisonMode mode) => CompareImplicitLengthNotCAndNotZ(left, right, mode);
```

Comment:
What does `C` and `Z` mean?

Reply:
> What does C and Z mean?
> What does O mean? Similiarly, what does the S suffix mean below?

They all represent the corresponding flags in the [FLAGS register](https://en.wikipedia.org/wiki/FLAGS_register) of x86/x64 CPUs.

>  don't use single letter abbreviations for prefixes/suffixes in names.

I agree, but now we have some similar APIs that "use single letter abbreviations", such as `Sse41.TestC`, `Avx.TestZ`, etc. 
Do you have any suggestion to improve the names?

Reply:
I assume `C` == `Carry`, `Z` == `Zero`, `O` == `Overflow`, and `S` == `Sign`?  Can we spell out the words?  Or even better, if we could come up a more descriptive name of what the method does, that would be great.

Reading https://software.intel.com/sites/landingpage/IntrinsicsGuide/#!=undefined&text=_mm_cmpestr&expand=814,813,814,813, the only difference between `S` and `Z` that I can tell is which argument to check for a null character:

```
Compare packed strings in a and b with lengths la and lb using the control in imm8, and returns 1 if any character in a was null, and 0 otherwise. 
```
vs.
```
Compare packed strings in a and b with lengths la and lb using the control in imm8, and returns 1 if any character in b was null, and 0 otherwise. 
```

It seems the name should indicate/describe the differences, instead of using `SFlag` or `ZFlag`, and then forcing the caller to look up what `SFlag` means.